src/main/java/hlf/java/rest/client/controller/FabricClientController.java

```diff
@@ -224,4 +225,32 @@ public ResponseEntity<EventAPIResponseModel> getTransactionByBlockNumber(
     return transactionFulfillment.readTransactionEventByBlockNumber(
         blockNumber, networkName, transactionId, chaincode, eventType);
   }
+
+  /**
+   * The REST Endpoint for supporting write transactions to ledger by accepting a Payload that can
+   * consists of multiple Private Data Collection details as well as multiple Public data.
+   *
+   * @param channelName String channel name
+   * @param chaincodeName String chaincode name
+   * @param functionName String function name in chaincode
+   * @param payload a Request body of kind {@link MultiDataTransactionPayload}
+   * @return responseEntity ResponseEntity Transaction Response
+   */
+  @PostMapping(value = "/invoke_transaction/multi_data")
+  public ResponseEntity<ClientResponseModel> invokeTransaction(
+      @RequestHeader("channel") @Validated String channelName,
+      @RequestHeader("chaincode") @Validated String chaincodeName,
+      @RequestHeader("function") @Validated String functionName,
```

Comment:
While sending transaction we have used same fields as `RequestHeader` in one api and `RequestParam` in another, should these parameters be part of `RequestHeader`, as I think these are `RequestParam` candidates

Reply:
Sorry, I didnt quite understand your question. Are you proposing to change these to RequestParams? I set these as requestHeaders since I see the existing Write APIS accept these params as headers, specifically invoke_transaction (Althogh read APIs accept these as query params)

Reply:
`/write_transaction` uses these as RequestParams and `invoke_transaction` is using them as RequestHeader, so for writing to ledger we are using parameters as params in one case and header in another, that is why I raised this question, that since this a new api, what should be the correct way to pass these values

Reply:
Understood. In that case lean towards the usage of `RequestParam` over `RequestHeaders`, these parameters are more qualified to be set as `RequestParam`.

Reply:
[nitpick] Not related to this PR. The `write_transaction` should be reworded as `/trasaction/simulate`. Can this be done by adding a new controller handler and marking the current API/controller handler for deprecation?